### DISCUSSION TITLE
Can I implement EvalEx by Gradle in my Android calculator app I'm currently developing? Is there any problem related with copyright?

### DISCUSSION THREAD

**Discussion by Megabyte94** [ezylang/EvalEx#288] [Q&A] [answered]
I am developing an Android calculator app that implements EvalEx by Gradle and app uses EvalEx for calculating expressions written by the user.

But I don't know if there would be any copyright problem with the creator of EvalEx if I upload my app in Google Play Store using EvalEx for calculating expressions' results.

I would like to receive a reply according to this. And if the creator of EvalEx could be the person who replies, I would be very grateful. 

Cheers.

**Comment by uklimaschewski** [comment, marked as answer]
EvalEx is published under the MIT license.
Here is a good FAQ that will answer your questions:
https://www.tawesoft.co.uk/kb/article/mit-license-faq

**Reply by Megabyte94** [reply]
Thank you very much!
